app/views/docs/permissions.phtml

```diff
@@ -123,10 +125,14 @@ client
 const databases = new Databases(client, '[DATABASE_ID]');
 
 let promise = databases.createDocument(
-    '[COLLECTION_ID]',
-    {'actorName': 'Chris Evans', 'height': 183},
-    ['role:all'], // Anyone can view this document
-    ['user:5c1f88b42259e'] // You can only grant permissions you own, this must be the current user UID
+    '[COLLECTION_ID]', // Collection ID
+    {'actorName': 'Chris Evans', 'height': 183}, // Data
+    [ // Permissions
+        Permission.read(Role.any()), // Anyone can view this document
+        Permission.edit([Role.team("writers"), Role.team("admin")]), // Only writers and admins can edit this document
+        Permission.delete(Role.user("5c1f88b42259e")) 
```

Comment:
I think this is supposed to be `Permission.delete(Role.team("administrators"))`. We mentioned admins can delete, but in this example we allow delete to user twice.

Reply:
Also missing comment and comma.

Reply:
Is there a reason why we put both `edit` permissions into one array, but with `delete` we separated it into two methods? If on purpose, can we mention it somehow, so people know both works and it's up to them?